packages/logger/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@empathyco/x-logger",
+  "version": "1.1.0",
+  "description": "Empathy simple logger",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./types/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc && tsc --project tsconfig.esm.json",
+    "build:watch": "tsc --watch",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://cbn-javieri@bitbucket.org/colbenson/logger.git"
+  },
+  "keywords": [
```

Comment:
We might want to update this keywords, this project is not related in any way with these `keywords`.

Suggestions:
- `logger`
- `logging`
- `tools`